frontend/pages/artist/_itemId/index.vue

```diff
@@ -174,6 +292,24 @@ export default Vue.extend({
       })
     ).data.Items;
 
+    const singles = discography?.filter(
+      (album: BaseItemDto) =>
+        (album?.RunTimeTicks || album?.CumulativeRunTimeTicks || 0) <=
+        albumBreakpoints.singleTicksMaxLength
+    );
+    const eps = discography?.filter(
+      (album: BaseItemDto) =>
+        (album?.RunTimeTicks || album?.CumulativeRunTimeTicks || 0) >
+          albumBreakpoints.singleTicksMaxLength &&
+        (album?.RunTimeTicks || album?.CumulativeRunTimeTicks || 0) <=
+          albumBreakpoints.epTicksMaxLength
+    );
+    const albums = discography?.filter(
+      (album: BaseItemDto) =>
+        (album?.RunTimeTicks || album?.CumulativeRunTimeTicks || 0) >
+        albumBreakpoints.epTicksMaxLength
+    );
+
```

Comment:
Feels to me like this business logic should live in a separate ts file, perhaps? Not an expert in either Vue or how this project is using it, just interested Jellyfin user. :)

Reply:
Not a Vue expert either, but it feels more like display logic than business logic to me.